src/styles/blocks/card.scss

```diff
@@ -0,0 +1,123 @@
+.card {
+  @extend %flexAlignCenter;
+  flex-direction: column;
```

Comment:
Illia said that we don't need flex aligning here, because almost all elements are blocks and take all available width (except image, you should make it block manualy)

Reply:
@KuharchukDmytro  Disagree with you. This is not a bug, this is an alternate solution